assets/config.yaml

```diff
@@ -6,7 +6,7 @@ namespaces:
         description: Connect to a favourite wifi network.
         interactive: false
         tags:
-        - netowrk
+          - netowrk
```

Comment:
@coderabbitai correct the typo

Reply:
Create a PR with a change fixing this.

Reply:
You have not created a branch as far as I can see. Why can you not create a PR? You did so with the docstrings.